### PR TITLE
ACS-8719: add response completeness field to

### PR DESCRIPTION
**Pull request by Pawel-608**

### CODE REVIEW

File: docs/acs-private-apis.md [hunk 1]
```diff
@@ -137,35 +137,24 @@ Get answer for a question.
 * Response:
     * Status: 200
     * Content-Type: `application/json`
-    * Entry object (wrapped with list/pagination information):
```
Comment (krdabrowski) [inline docs/acs-private-apis.md:140]: There's only one answer for a question?

Comment (krdabrowski) [reply docs/acs-private-apis.md:140]: In path `questions/{questionId}/answers` there shouldn't be also present `/-default-` ID?

**Review by krdabrowski — APPROVED** [review]
I'm not entirely convinced with this change: list -> entry 🤔 But in the other hand I don't have any serious cons (except required ID in path), so I'm fine with that. WDYT @mpichura ?

File: docs/acs-private-apis.md (hunk 1 above)
Comment (Pawel-608) [reply docs/acs-private-apis.md:140]: yep, from what I saw in code there is just one answer but there are multiple "Reference" inside

Comment (Pawel-608) [reply docs/acs-private-apis.md:140]: fixed path

**Review by mpichura — APPROVED** [review]
The change looks okay to me. 
Since HXI returns only one answer, I think we shouldn't wrap it around pagination.

**Pull request merged**